src/typing_manager.ts

```diff
@@ -218,33 +215,11 @@ export class TypingManager implements Disposable {
         await this.onSendCommand(key);
     };
 
-    public onEnableCommand = (mode: "enable" | "disable" | "toggle"): void => {
-        switch (mode) {
-            case "enable":
-                this.neovimEnable = true;
-                break;
-            case "disable":
-                this.neovimEnable = false;
-                break;
-            case "toggle":
-                this.neovimEnable = !this.neovimEnable;
-                break;
-        }
-
-        if (this.neovimEnable) {
-            this.client.command("stopinsert");
-        } else {
-            this.client.command("startinsert");
```

Comment:
This was supposedly an important part of https://github.com/vscode-neovim/vscode-neovim/pull/1033 for "preserving the mode when switching between windows". Is that use-case still addressed?

Reply:
I didn't read the PR in detail. These commands need to be executed manually, and I didn't see any use of this command when searching globally. I don't know how it was originally used, but I think it is redundant, and the names of these three commands are easily misleading. We can discuss this later when someone gives feedback. I prefer to provide one action instead of three commands, and it should also be possible to use BufEnter directly.